<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresentationModeWindow</class>
 <widget class="QMainWindow" name="PresentationModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">PresentationWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="centralAreaStackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QStackedWidget {
    background-color: #000000;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="capture::common::VideoWidget" name="videoWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
	background-color: black;
}</string>
       </property>
      </widget>
      <widget class="capture::presentation::StageViewerMirror" name="stageViewerMirror">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>capture::common::VideoWidget</class>
   <extends>QWidget</extends>
   <header>common/video_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>capture::presentation::StageViewerMirror</class>
   <extends>QWidget</extends>
   <header>presentation/stage_viewer_mirror.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
